File: FertiWeb/src/main/java/dominio/Parcela.java
package dominio;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class Parcela {

	private Long codigoParcela;
	private Long codigoCultivoSembrado;
	private Long codigoLugar;
	private double area;
	private Date fechaSiembra;

}

File: FertiFront/src/app/clases_dominio/parcela.ts
import { TipoCultivo } from './tipo-cultivo';
import { Lugar } from './lugar';

export class Parcela {
    public codigoParcela: number;
    public lugar: Lugar;
    public tipoCultivo: TipoCultivo;
    public area: number;
    public fechaSiembra: Date;
}
